docs/tutorials/using_apis/connectivity/light-control.md

```diff
@@ -63,95 +99,69 @@ An example of the `mbed_app.json` file:
+1. Connect the board to your computer, and power on the board.
+1. Drag and drop the binary to the board (shown as storage device on the computer) to program the application.
 
 ## Update the firmware of the border router
+<!--do I have to do this? do I have to do this at this point in the process, or can I put it as one of the requirements that should have been listed earlier in the doc?-->
 
-This example supports the following border router:
-
-[Nanostack-border-router](https://github.com/ARMmbed/nanostack-border-router).
+This example supports the [Nanostack-border-router](https://github.com/ARMmbed/nanostack-border-router).
 
-The border router supports static and dynamic backhaul configuration. The static configuration is good for testing, but the dynamic one works if your network infrastructure is supplying an IPv6 address. Make sure that you use the appropiate mode.
+The border router supports static and dynamic backhaul configuration. The static configuration is good for testing, but the dynamic one works <!--this phrasing is misleading, I think. why are testing and dynamic allocation contradictory?--> if your network infrastructure supplies an IPv6 address. Make sure that you use the appropriate mode.<!--is this what I'm supposedly updating?-->
 
-Remember to connect the Ethernet cable between the border router and your router. Then power on the board.
+Remember to connect the Ethernet cable between the border router and your router<!--router and router? router and board?-->. Then power on the board.<!--it was already powered on to flash it. did I disconnect it from the computer at some point? did I power it off?--><!--is this really part of "update your router"? won't it be equally valid if my border router was already correct?-->
 
-## Testing
+## Testing with border router
 
-By default, the application is built for the LED control demo, in which the device sends a multicast message to all devices in the network when the button is pressed. All devices that receive the multicast message will change the LED status (red LED on/off) to the state defined in the message. Note that the Thread devices can form a network without the existence of the border router. The following applies only to the case when the border router is set up.
+By default, the application is built for the LED control demo, in which the device sends a multicast message to all devices in the network when the button is pressed. All devices that receive the multicast message will change the LED status (red LED on/off) to the state defined in the message.<!--so do I need a second application that I can flash to other devices that I connect to the network? or do I just need more devices using the same application? this should be explained at the beginning--> Note that the Thread devices can form a network without the existence of the border router. The following applies only to the case when the border router is set up.
```

Comment:
@SeppoTakalo @jarlamsa Can you answer this?

Reply:
As the paragraph is about testing with border router, then yes, one device needs to be a border router and have a separate application `nanostack-border-router` flashed and configured properly.